projects/igniteui-angular/src/lib/grids/common/random.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Use the function to get a random UUID string when secure context is not guaranteed making crypto.randomUUID unavailable.
+ * @returns A random UUID string.
+ */
+export function getUUID() {
+    if (typeof crypto.randomUUID === "function") {
```

Comment:
Minor nit, but per the repo settings we use single quotes (goes for the rest in the file obv). Also do set the return type instead of the jsdoc tag since we do use TS. You can be even more specific and type it exactly like `randomUUID` is with the pattern if you'd like.
```suggestion
 * Get a random UUID string. Uses `crypto.randomUUID` if available or a fallback implementation when not in secure context.
 */
export function getUUID(): string {
    if (typeof crypto.randomUUID === 'function') {
```

Reply:
PS: Don't mind this getting release and switched later too :)